Os/Stub/ConditionVariable.hpp

```diff
@@ -38,6 +38,9 @@ class StubConditionVariable : public ConditionVariableInterface {
     //! \brief get handle
     ConditionVariableHandle* getHandle() override;
 
+    //! \brief assignment operator is forbidden
+    virtual ConditionVariableInterface& operator=(const ConditionVariableInterface& other) override = delete;
```

Comment:
Note: using `virtual` on a line marked `override` is redundant.  Not a blocking issue.